CHANGES.md

```diff
@@ -8,6 +8,52 @@ Rust image aims to be a pure-Rust implementation of various popular image format
 
 ## Changes
 
+### Version 0.23.0-preview.0
+
+This major release intends to improve the interface with regards to handling of
+color format data for both decoding and encoding. This necessitated many
+breaking changes anyways so it was used to improve the compliance to the
+interface guidelines such as outstanding renaming.
+
+WIP: This preview version is intended to allow an evaluation and feedback on
+the new interface. It is not yet perfect with regards to color spaces but it
+was designed mainly as an improvement over the current interface with regards
+to in-memory color formats, first. We'll get to color spaces in a later major
+version.
+
+- Heavily reworked `ColorType`:
+  - This type is now used for denoting formats for which we support operations
+      on buffers in these memory representations. Particularly, all channels in
+      pixel types are assumed to be sized in units of bytes.
```

Comment:
"assumed to be sized in units of bytes" sounds like RGB8 might be 8 bytes for each of the red green and blue channels. "assumed to be an integer number of bytes" might be more clear

Reply:
I wanted to capture a relation to Rust's type system here but the wording trailed off into 'clever' instead of 'understandable' :grinning: